recording/lib/metrics.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Membrane.RTC.Engine.Endpoint.Recording.Metrics do
+  @moduledoc """
+  Defines list of metrics, that Reporter instance can aggregate by listening on events emitted in RTC Engine.
+  Suggested Reporter implementation is `Membrane.TelemetryMetrics.Reporter` from
+  [`membrane_telemetry_metrics`](https://github.com/membraneframework/membrane_telemetry_metrics).
+  You can see usage example in [`membrane_videoroom`](https://github.com/membraneframework/membrane_videoroom).
+
+  See also `Metrics` guide for more details and example report that you can obtain with
+  `Membrane.TelemetryMetrics.Reporter`.
+  """
+
+  require Membrane.TelemetryMetrics
+
+  @upload_event [Membrane.RTC.Engine.Endpoint.Recording, :S3]
+
+  @spec metrics() :: [Telemetry.Metrics.t()]
+  def metrics() do
+    [
+      Telemetry.Metrics.counter(
+        "recording.s3-upload",
+        event_name: @upload_event,
+        tags: [:status]
```

Comment:
We miss the `measurement`

Reply:
In the counter metric, you shouldn't pass the `measurement` parameter.